### PR TITLE
add queries to the GET /articles endpoint and test suites

### PR DESCRIPTION
**Pull request by LukeHarrisonDev**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -92,6 +92,53 @@ describe("/api/articles", () => {
...
+                expect(body).toEqual({message: "Bad request"})
+            })
+        })
+        test("?order=: Responds with all article objects ordered by the given 'order' query", () => {
+            return request(app)
+            .get("/api/articles?order=asc")
+            .expect(200)
+            .then(({body})=> {
+                expect(body.articles).toHaveLength(13)
+                expect(body.articles).toBeSortedBy('created_at')
```
Comment (nc-niamh-smith) [inline __tests__/app.test.js:119]: I think because of the test, I'd expect to see the order asc be true as well :)

**Review by nc-niamh-smith — COMMENTED** [review]
Nice work! 🎉 I've asked a couple of questions re tests - just a couple of things to add to a test or two, nothing too wild! :)

File: __tests__/app.test.js (hunk 1 above)
Comment (LukeHarrisonDev) [reply __tests__/app.test.js:119]: @nc-niamh-smith 
Cheers Niamh, Isn't it ascending by default?

Comment (nc-niamh-smith) [reply __tests__/app.test.js:119]: Yessss but it'd be quite nice to have it explicit in case anyone looking wasn't 100%! :)

Comment (LukeHarrisonDev) [reply __tests__/app.test.js:119]: Thats fair. Cheers :)

**Pull request merged**